README.md

```diff
@@ -110,7 +110,7 @@ AutoMQ: A Cloud-Native fork of Kafka by separating storage to S3
     <td>In hours/days</td>
     <td>In hours</td>
     <td>In seconds</td>
-    <td>In hours</td>
+    <td>In seconds</td>
```

Comment:
We can use "In hours/In seconds (Enterprise Only)" here. What do you think?

Reply:
Real users of any log are likely using enterprise anyway, so best to compare what they are most likely to be using. i.e. these things don't matter to hobbyists who would be using the community edition, so it's most fair to fit the comparison to the target users.

It's unlikely that someone is deciding between the two solutions on these two lines alone.